var/spack/repos/builtin/packages/ilmbase/package.py

```diff
@@ -10,10 +10,17 @@ class Ilmbase(AutotoolsPackage):
     """OpenEXR ILM Base libraries (high dynamic-range image file format)"""
 
     homepage = "http://www.openexr.com/"
-    url      = "http://download.savannah.nongnu.org/releases/openexr/ilmbase-2.2.0.tar.gz"
+    url      = "https://github.com/openexr/openexr/releases/download/v2.3.0/ilmbase-2.3.0.tar.gz"
```

Comment:
why did you change the url?

Reply:
Upstream have changed their distribution method. Seems that new versions are coming from github not savannah.  (See http://www.openexr.com/downloads and http://mirror.netcologne.de/savannah/openexr/ the latter has the new versions missing)

Reply:
Does github provide the old versions? It would be nice not to have a url in every version...

Reply:
They do but it still requires some faffing about with custom URLS.  Spack detects back to 1.7.1 from a url= but prior versions to that are not detected because the file naming scheme completely changes.

How far back in versions should we support?